cosmoz-treenode-navigator.js

```diff
@@ -141,13 +144,26 @@
 				value: 1
 			}
 		},
-		_computeDataPlane: function (searching, inputValue, renderedLevel, openNodeLevelPathParts, data) {
+
+		_computeDataPlane: function (searching, inputValue, renderedLevel, openNodeLevelPathParts, tree) {
 			if (searching) {
-				return this.searchAllBranches(inputValue, openNodeLevelPathParts, renderedLevel, data);
+				return tree.getNodeByProperty(this.comparisonProperty, inputValue, false, true, renderedLevel);
 			}
 			return renderedLevel;
 		},
 		/**
+		 * A hook to manipulate the default tree class
+		 * of Cosmoz.TreeBehavior
+		 */
+		_getTreeClass: function (treeClass) {
+			treeClass.prototype.afterFound = function (node, propertyName) {
```

Comment:
We can. (just a tiny little bit less efficient but no big deal)
 I'm just used to such hook methods I know from js-data.